web_app/main_code_basis.py

```diff
@@ -0,0 +1,15 @@
+from learn_python.AVITO_CLON.Learn_pyhton_site_clon.web_app import app
```

Comment:
в модуле web_app нет экземпляра приложения, только фабрика, которая его возвращает
```suggestion
from web_app import create_app
```

Reply:
У меня запускается только если передлать на from __init__ import create_app - так правильно?